### DISCUSSION TITLE
v1.10.10-lts

### DISCUSSION THREAD

**Discussion by wanghe-fit2cloud** [1Panel-dev/1Panel#5369] [Announcements]
## 新增功能

* 【数据库】支持多版本 Redis 数据库的切换功能。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5001
* 【数据库】支持添加远程 Redis 数据库功能。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5014
* 【数据库】支持创建 Redis 数据库的快速命令功能。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5018
* 【容器】新增清理构建缓存功能。 by @john1298308460 in https://github.com/1Panel-dev/1Panel/pull/4978
* 【主机】tar.gz 类型文件压缩和解压功能支持设置密码。 by @john1298308460 in https://github.com/1Panel-dev/1Panel/pull/5258
* 【工具箱】新增 FTP 功能。by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5039
* 【面板设置】新增代理服务器设置功能。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5189
* 【系统】支持为备份类型计划任务以及快照功能设置压缩密码。 by @john1298308460 in https://github.com/1Panel-dev/1Panel/pull/5258
* 【X-Pack】新增网站监控功能。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5038
* 【X-Pack】支持黑金主题。 by @LIlGG in https://github.com/1Panel-dev/1Panel/pull/5015

## 功能优化

* 【应用商店】应用升级时支持对比 Compose 文件，以确保兼容手动修改的配置。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5274
* 【应用商店】优化应用查询接口的响应速度。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5202
* 【应用商店】优化应用同步接口的响应速度。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5241
* 【网站】添加 Node.js 类型的运行环境时可选择使用 pnpm 作为包管理器。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5225
* 【网站】网站目录快速跳转至 index 目录。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5025
* 【网站】创建运行环境和静态网站时支持创建 FTP 用户。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5076
* 【数据库】添加远程数据库时增加名称校验。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5222
* 【数据库】优化 Redis 数据库页面的样式。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5229
* 【数据库】Redis 数据库在安装时不再要求强制设置密码。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5239
* 【主机】收藏夹悬浮框显示文件夹全路径。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5097
* 【工具箱】缓存清理功能支持容器垃圾清理。 by @john1298308460 in https://github.com/1Panel-dev/1Panel/pull/5024
* 【面板设置】删除快照页面增加是否删除备份文件的选项。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5237
* 【网站】网站日志使用主配置文件参数。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5255
* 【系统】演示环境禁用容器终端功能。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5235
* 【系统】提升移动端登录时勾选开源许可复选框的用户体验。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5259
* 【系统】取消右侧抽屉页面的 ESC 按钮关闭功能。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/4979
* 【系统】定时清理 cache 缓存以优化性能。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5226
* 【系统】优化文件分段上传的用户体验。 by @qwenode in https://github.com/1Panel-dev/1Panel/pull/5123
* 【系统】优化安全入口页面的样式特征。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5159
* 【X-Pack】优化 CDN IP 的获取方式。

## 问题修复

* 【概览】修复了部分磁盘类型状态显示不完整的问题。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5190
* 【应用商店】修复了本地应用升级后版本信息未显示的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5261
* 【应用商店】修复了应用安装过程中切换版本时校验条件未同步更改的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5205
* 【应用商店】修复了部分场景下应用安装失败后未正确显示的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5295
* 【网站】修复了自签证书关闭页面后未刷新的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5282
* 【网站】修复了编辑证书页面未显示证书秘钥算法的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5301
* 【网站】修复了 PHP 运行环境镜像构建中可以被删除的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5304
* 【网站】修复了删除网站后备份文件未同步删除的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5309
* 【网站】修复了清空伪静态配置报错的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5179
* 【网站】修复了使用本地 PHP 环境启动、停止网站后出现配置错误的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5198
* 【网站】修复了网站域名带有多个端口时，快速跳转导航链接错误的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/4998
* 【数据库】修复了远程 MariaDB 数据库备份失败的问题。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5221
* 【数据库】修复了远程数据库列表按名称排序显示错误的问题。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5257
* 【主机】修复了部分场景 SSH 登录日志解析失败的问题。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5081
* 【主机】修复了文本编辑器在打开 GBK 编码文件时出现乱码的问题。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5096
* 【主机】修复了非文本文件打开报错的问题。 by @zhengkunwang223 in https://github.com/1Panel-dev/1Panel/pull/5203
* 【计划任务】修复了计划任务执行周期解析错误的问题。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5119
* 【计划任务】修复了计划任务列表和报告页面上次执行时间显示不一致的问题。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5230
* 【系统】修复了系统升级过程中数据库文件备份异常的问题。 by @ssongliu in https://github.com/1Panel-dev/1Panel/pull/5120
* 【X-Pack】修复了低版本 OpenResty 打开 WAF 概览页时报错的问题。
* 【X-Pack】修复了清空 WAF 日志时未同步删除首页统计数据的问题。
* 【X-Pack】修复了网站防篡改状态切换异常的问题。

<hr /><em>This discussion was created from the release <a href='https://github.com/1Panel-dev/1Panel/releases/tag/v1.10.10-lts'>v1.10.10-lts</a>.</em>

**Comment by mobeicanyue** [comment]
为什么 Github 首页收不到 release 推送
